### DISCUSSION TITLE
data types inheritance

### DISCUSSION THREAD

**Discussion by triongroup** [pimcore/pimcore#9201] [Q&A / Support] [answered]
Hi!
I'm confused about academy's question "Which data types do support data inheritance?"
After I've listened to all the videos and read all the documentation about this chapter I still get error on this question. I tried to answer with Blocks and Object Bricks as documentation mentioned, but still get an error. 
<img width="466" alt="Снимок экрана 2021-05-24 в 15 06 01" src="https://user-images.githubusercontent.com/13614676/119345474-9b86ee80-bca1-11eb-962d-998446f4aef5.png">
Why my answer is wrong? Please help.

**Comment by fashxp** [comment, marked as answer]
All data types except field collection do support data inheritance.

**Reply by triongroup** [reply]
Many thanks! You really help me!
